src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,40 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+    Animal.alive.push(this);
+  }
```

Comment:
this.health equals 100 always
```suggestion
  constructor(name) {
    this.name = name;
    this.health = 100;
    Animal.alive.push(this);
  }
```